libethereum/EthereumCapability.cpp

```diff
@@ -629,7 +629,7 @@ void EthereumCapability::onDisconnect(NodeID const& _peerID)
 bool EthereumCapability::interpretCapabilityPacket(
     NodeID const& _peerID, unsigned _id, RLP const& _r)
 {
-    auto& peer = m_peers[_peerID];
+    auto& peer = this->peer(_peerID);
```

Comment:
```suggestion
    auto& peer = peer(_peerID);
```

Reply:
I intentionally went with `this->peer ` since otherwise I get a compiler error:
```
C:\Users\nilse\Documents\Code\aleth\libethereum\EthereumCapability.cpp(632,22): error C3536:  'peer': cannot be used before it is initialized [C:\Users\nilse\Documents\Code\aleth\build\libethereum\ethereum.vcxproj]
C:\Users\nilse\Documents\Code\aleth\libethereum\EthereumCapability.cpp(632,18): error C2064:  term does not evaluate to a function taking 1 arguments [C:\Users\nilse\Documents\Code\aleth\build\libethereum\ethereum.vcxproj]
C:\Users\nilse\Documents\Code\aleth\libethereum\EthereumCapability.cpp(654,46): error C2664:  'void dev::eth::EthereumPeerObserverFace::onPeerStatus(const dev::eth::EthereumPeer &)': cannot convert argument 1 from 'int' to 'const dev::e
th::EthereumPeer &' [C:\Users\nilse\Documents\Code\aleth\build\libethereum\ethereum.vcxproj]
```

Reply:
Rename `peer` variable then